### DISCUSSION TITLE
VDH Lizenz Kaufen ohne Kreditkarte

### DISCUSSION THREAD

**Discussion by Sven941** [aclap-dev/video-downloadhelper#858] [Help and Q&A]
### Your question

Hallo zusammen,
wie komme ich an die Lizenz ohne Kreditkarte? Ich habe nämlich keine.
Danke euch schon mal.

### Addon & CoApp version and details

Hallo zusammen,
wie komme ich an die Lizenz ohne Kreditkarte? Ich habe nämlich keine.
Danke euch schon mal.

**Comment by jcfrog** [comment]
hi,
sorry there is no other way than using Stripe payment platform.
